OLEDDisplay.h

```diff
@@ -201,10 +201,10 @@ class OLEDDisplay : public Print {
     void normalDisplay(void);
 
     // Set display contrast
-    void setContrast(char contrast);
+	void setContrast(char contrast, char precharge = 241, char comdetect = 64);
 
     // Turn the display upside down
-    void flipScreenVertically();
+    void flipScreenVertically(bool mirror = false, bool turn = false);
 
     // Write the buffer to the display memory
     virtual void display(void) = 0;
```

Comment:
Hi and thanks @lorol 

I'd rather have a PR only for the glyphEditor.html file changes, can you do that ?

Also, do you know why the full file glyphEditor.html looks new here, there is no diff with mine ?  I'm somewhat puzzled...

Reply:
Yeah, a mess ... Sorry. I have no experience with this stuff ...
I will thy to re PR from you with committed changes  to glyphEditor.html

Reply:
No problem :) but I think you need to fork my repo, to submit your modifications of the glyphEditor.html file :)

Reply:
Oops. OK will try this way

Reply:
I somehow cannot fork yours because once I forked squix' - I think.
Anyway ... wi'll see later.

Reply:
Ah sorry I never tried that actually :(
You probably would need to fork mine in a different directory on your disk.

Reply:
It looks I have to delete entirely my existing fork from GitHub and get a new one from yours. 
Not sure what will happen with any related previous history, map, discussions, if I do so ...

Anyway ...

BTW check in your copy function:
            ` let tableOffset = charNumber + parseInt(document.getElementById('code').value) - 2 ;`
I think it should be: 
             `let tableOffset = charNumber - parseInt(document.getElementById('code').value);`

Reply:
Hi again,
I just tried to fork your repo, and you are right, it does not seem to be possible because I already forked squix78's repo. I didn't know that.
Do not delete anything, I'll just include your modifications, it'll be simpler and you won't risk losing anything.
The formula of the copy function was not good indeed, the start code needs to be subtracted, not added.

Reply:
Ok it's done. I didn't keep your change of adding the character itself in the caption, since I'm not designing character fonts but icons, I may use codes that do not match displayable characters.
Let me think it through, since it has value when using it for actual character fonts... anyway this caption initialization should be in a method since it's done several time... char code could be checked for "displayability" there.
Anyway thanks for the nice improvement with file loading and saving.
I'll close this PR since it's no longer relevant.